src/main/java/homework_2/traffic_light/utils/MessageType.java

```diff
@@ -0,0 +1,25 @@
+package homework_2.traffic_light.utils;
+
+public enum MessageType {
+    ERROR_MESSAGE_NEGATIVE_NUMBERS("You should enter only positive numbers or 0."),
+    ERROR_MESSAGE_INCORRECT_FORMAT("You entered time in wrong format."),
+    ERROR_MESSAGE_EXCEED_LIMIT("Please do not exceed the time limits. There are only 86399 seconds or 23:59:59 in a day!"),
```

Comment:
Opt: If possible, please use exception messages as in homework chat

Reply:
done